mlflow/models/signature.py

```diff
@@ -147,11 +165,14 @@ def infer_signature(
     :param model_input: Valid input to the model. E.g. (a subset of) the training dataset.
     :param model_output: Valid model output. E.g. Model predictions for the (subset of) training
                          dataset.
+    :param parameters: Valid parameters for inference. E.g. a dictionary of params used
+                       for inference.
```

Comment:
A bit confused by the use of "e.g." here (which means "for example"). The params *must* be a dictionary, right?

Reply:
Can we also include examples of parameter names / values and include an example code snippet demonstrating how these are saved? Additionally, can we explain that these parameters can be set when calling the model at inference time, e.g. by passing `params` to the pyfunc `predict()` method?